src/pages/signin/SignInPage.js

```diff
@@ -78,7 +82,9 @@ class SignInPage extends Component {
         // - AND an account did not exist or is not validated for that login
         const showResendValidationLinkForm = this.props.credentials.login && !validAccount;
 
-        const welcomeText = this.props.translate(`welcomeText.${showPasswordForm ? 'phrase4' : 'phrase1'}`);
+        const welcomeText = showResendValidationLinkForm
+            ? ''
+            : this.props.translate(`welcomeText.${showPasswordForm ? 'phrase4' : 'phrase1'}`);
```

Comment:
Eep, another thing that is not your fault, but would be nice to update while we are here. `phrase1` and `phrase4` are not very descriptive names. I think this would be easier to read if it said something like:

```js
this.props.translate(`welcomeText.${showPasswordForm ? 'welcomeBack' : 'welcome'}`);
```